### DISCUSSION TITLE
Buttons settings and Cursor and acceleration control

### DISCUSSION THREAD

**Discussion by kostazzz** [Caldis/Mos#590] [Ideas]
Settings for the side buttons and additional mouse cursor acceleration control are **very lacking.** Is it possible to wait for the next releases?

Thank you so much for your work, without your application it is impossible to use a mouse on a mac at all. :)

**Comment by HVR88** [comment]
This looks like it's strictly a mouse-wheel program. The smooth scrolling and especially INERTIA (duration) make this a dream for using third-party mice.

If you want button assignments, may I suggest USB Overdrive (also handles acceleration) or Better Touch Tool (buttons only) or Mac Mouse Fix (buttons and scrolling, but make sure to disable scrolling so Mos can keep working)
